client/extensions/woocommerce/app/order/order-customer/create.js

```diff
@@ -0,0 +1,182 @@
+import { getOrderWithEdits } from 'woocommerce/state/ui/orders/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import SectionHeader from 'components/section-header';
+
+class OrderCustomerInfo extends Component {
+	static propTypes = {
+		editOrder: PropTypes.func.isRequired,
+		orderId: PropTypes.oneOfType( [
+			PropTypes.number, // A number indicates an existing order
+			PropTypes.shape( { id: PropTypes.string } ), // Placeholders have format { id: 'order_1' }
+		] ).isRequired,
+		siteId: PropTypes.number.isRequired,
+	};
+
+	state = {
+		showDialog: false,
+	};
+
+	updateAddress = ( type = 'billing' ) => {
+		const { siteId, order } = this.props;
+		return address => {
+			const { copyToShipping = false, ...newAddress } = address;
+			if ( siteId ) {
+				this.props.editOrder( siteId, { id: order.id, [ type ]: newAddress } );
+				if ( copyToShipping && 'billing' === type ) {
+					this.props.editOrder( siteId, { id: order.id, shipping: newAddress } );
+				}
+			}
+		};
+	};
+
+	toggleDialog = type => {
+		return () => {
+			this.setState( { showDialog: type } );
+		};
+	};
+
+	renderDialogs = () => {
+		const { billing, shipping } = this.props.order;
+		return [
+			<CustomerAddressDialog
+				key="dialog-billing"
+				address={ billing }
+				closeDialog={ this.toggleDialog( false ) }
+				isBilling
+				isVisible={ 'billing' === this.state.showDialog }
+				updateAddress={ this.updateAddress( 'billing' ) }
+			/>,
+			<CustomerAddressDialog
+				key="dialog-shipping"
+				address={ shipping }
+				closeDialog={ this.toggleDialog( false ) }
+				isVisible={ 'shipping' === this.state.showDialog }
+				updateAddress={ this.updateAddress( 'shipping' ) }
+			/>,
+		];
+	};
+
+	renderBilling = ( address = {} ) => {
+		const { translate } = this.props;
+		if ( every( address, isEmpty ) ) {
+			return (
+				<Button
+					className="order-customer__add-link"
+					onClick={ this.toggleDialog( 'billing' ) }
+					primary
+				>
+					{ translate( 'Add Billing Address' ) }
+				</Button>
+			);
+		}
+		return (
+			<Fragment>
+				<h4>{ translate( 'Address' ) }</h4>
+				<div className="order-customer__billing-address">
+					<p>{ `${ address.first_name } ${ address.last_name }` }</p>
+					<AddressView address={ getAddressViewFormat( address ) } />
+				</div>
+
+				<h4>{ translate( 'Email' ) }</h4>
+				<p>{ address.email }</p>
+
+				<h4>{ translate( 'Phone' ) }</h4>
+				<div className="order-customer__billing-phone">{ address.phone }</div>
+				<Button className="order-customer__add-link" onClick={ this.toggleDialog( 'billing' ) }>
+					{ translate( 'Edit Address' ) }
+				</Button>
+			</Fragment>
+		);
+	};
+
+	renderShipping = ( address = {} ) => {
+		const { translate } = this.props;
+		if ( every( address, isEmpty ) ) {
+			return (
+				<Button
+					className="order-customer__add-link"
+					onClick={ this.toggleDialog( 'shipping' ) }
+					primary
+				>
+					{ translate( 'Add Shipping Address' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Shipping address' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-new-order-creation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).